tests/topicsolved_test.php

```diff
@@ -40,9 +43,12 @@ public function setUp()
 
 		$this->user = $this->getMock('\phpbb\user', array(), array('\phpbb\datetime'));
 		$this->auth = $this->getMock('\phpbb\auth\auth');
+		$this->dispatcher = $this->getMockBuilder('\phpbb\event\dispatcher')
+			->disableOriginalConstructor()
+			->getMock();
```

Comment:
I think we can use phpBB's builtin method for mocking the dispatcher here as more shorthand, just like `main_controller_test` does:

```$phpbb_dispatcher = new \phpbb_mock_event_dispatcher();```